comment/views/comments.py

```diff
@@ -116,17 +121,22 @@ def post(self, request, *args, **kwargs):
 
 
 class ConfirmComment(View):
-    @staticmethod
-    def get(request, *args, **kwargs):
-        key = kwargs.get('key', None)
-        comment = get_comment_from_key(key)
+    email_service = None
 
-        if comment.why_invalid == CommentFailReason.BAD:
+    def get(self, request, *args, **kwargs):
+        key = kwargs.get('key', None)
+        temp_comment = get_comment_from_key(key)
+        if temp_comment.why_invalid == CommentFailReason.BAD:
             messages.error(request, EmailError.BROKEN_VERIFICATION_LINK)
-        elif comment.why_invalid == CommentFailReason.EXISTS:
+        elif temp_comment.why_invalid == CommentFailReason.EXISTS:
             messages.warning(request, EmailError.USED_VERIFICATION_LINK)
-
-        if not comment.is_valid:
+        if not temp_comment.is_valid:
             return render(request, template_name='comment/anonymous/discarded.html')
 
-        return redirect(comment.obj.get_url(request))
+        comment = temp_comment.obj
+        comment.save()
+        comment.refresh_from_db()
+        if settings.COMMENT_ALLOW_SUBSCRIPTION:
+            self.email_service = DABEmailService(comment, request)
+            self.email_service.send_notification_to_followers()
```

Comment:
using the signal will also allow us to remove this duplication here as the one used in different `view` functions.

Reply:
It's not valid as I commented above.